zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/GroupEntityRemovedHandler.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.handlers;
+
+import io.camunda.exporter.exceptions.PersistenceException;
+import io.camunda.exporter.store.BatchRequest;
+import io.camunda.webapps.schema.entities.usermanagement.EntityJoinRelation;
+import io.camunda.webapps.schema.entities.usermanagement.EntityJoinRelation.IdentityJoinRelationshipType;
+import io.camunda.webapps.schema.entities.usermanagement.GroupEntity;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.intent.GroupIntent;
+import io.camunda.zeebe.protocol.record.value.GroupRecordValue;
+import java.util.List;
+
+public class GroupEntityRemovedHandler implements ExportHandler<GroupEntity, GroupRecordValue> {
+
+  private final String indexName;
+
+  public GroupEntityRemovedHandler(final String indexName) {
+    this.indexName = indexName;
+  }
+
+  @Override
+  public ValueType getHandledValueType() {
+    return ValueType.GROUP;
+  }
+
+  @Override
+  public Class<GroupEntity> getEntityType() {
+    return GroupEntity.class;
+  }
+
+  @Override
+  public boolean handlesRecord(
+      final io.camunda.zeebe.protocol.record.Record<GroupRecordValue> record) {
+    return getHandledValueType().equals(record.getValueType())
+        && GroupIntent.ENTITY_REMOVED.equals(record.getIntent());
+  }
+
+  @Override
+  public List<String> generateIds(
+      final io.camunda.zeebe.protocol.record.Record<GroupRecordValue> record) {
+    return List.of(String.valueOf(record.getValue().getEntityKey()));
+  }
+
+  @Override
+  public GroupEntity createNewEntity(final String id) {
+    return new GroupEntity().setId(id);
+  }
+
+  @Override
+  public void updateEntity(final Record<GroupRecordValue> record, final GroupEntity entity) {
+    final GroupRecordValue value = record.getValue();
+    final var joinRelation =
+        new EntityJoinRelation()
+            .setName(IdentityJoinRelationshipType.MEMBER.getType())
+            .setParent(value.getGroupKey());
+    entity.setEntityKey(value.getEntityKey()).setJoin(joinRelation);
+  }
+
+  @Override
+  public void flush(final GroupEntity entity, final BatchRequest batchRequest)
+      throws PersistenceException {
+    batchRequest.delete(indexName, entity.getId());
```

Comment:
```suggestion
    batchRequest.deleteWithRouting(indexName, entity.getId());
```
We don't have that method yet, but it looks like it's easy to add.